climpred/tests/test_utils.py

```diff
@@ -3,36 +3,36 @@
 import numpy as np
 

```

Comment:
I think in `test_utils` there should be a check that all the aliases work. So import the alias dictionary and loop through it and check that you can run `get_metric_function` with the alias.